src/merkle.rs

```diff
@@ -0,0 +1,148 @@
+use sha3::{Digest, Sha3_256};
+
+type Hash = Vec<u8>;
```

Comment:
Hashes are fixed length, you can use [u8; 32]
```suggestion
type Hash = [u8; 32];
```

Reply:
Done